fr.openfisca.org/fr.openfisca.org.conf

```diff
@@ -28,31 +28,31 @@ server {
     location /api/v18 {
         # new url versioning shema
         proxy_pass http://localhost:6000/;
-        proxy_set_header Host $http_host;
+        proxy_set_header Host $http_host/api/v18;
         proxy_http_version 1.1;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     }
 
     location /api/v19 {
         # new url versioning shema
         proxy_pass http://localhost:8019/;
-        proxy_set_header Host $http_host;
+        proxy_set_header Host $http_host/api/v19;
         proxy_http_version 1.1;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     }
 
     location /api/v20 {
         # new url versioning shema
         proxy_pass http://localhost:8020/;
-        proxy_set_header Host $http_host;
+        proxy_set_header Host $http_host/api/v20;
         proxy_http_version 1.1;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     }
 
     location /api/v21 {
         # new url versioning shema
         proxy_pass http://localhost:8021/;
-        proxy_set_header Host $http_host;
+        proxy_set_header Host $http_host/api/v20;
```

Comment:
`proxy_set_header Host $http_host/api/v20;` => `proxy_set_header Host $http_host/api/v21;`

Reply:
That's normally error-prone, but `nginx` is flexible enough to let us refactor that so to avoid them:

```
location ~(/api/v21) {
  proxy_set_header Host $http_host$1;
}
```

Reply:
I get the following error : 
```
nginx: [emerg] "proxy_pass" cannot have URI part in location given by regular expression, or inside named location, or inside "if" statement, or inside "limit_except" block
```

Reply:
😢 

OK, that can be solved but is too complicated given what we're trying to solve.